analyses/1.Transfer_Sepsis_to_TB.ipynb

```diff
@@ -112,9 +112,9 @@
     "# Split train and test data\n",
     "\n",
     "data_dirs = util.get_data_dirs('../data')                                                      \n",
-    "train_dirs, test_dirs = util.extract_test_dirs(data_dirs, 'tb', sample_to_label)                                               \n",
+    "non_tb_dirs, tb_dirs = util.extract_dirs_with_label(data_dirs, 'tb', sample_to_label)                                               \n",
     "\n",
-    "train_dirs, tune_dirs = util.train_tune_split(train_dirs, 2)"
+    "train_dirs, tune_dirs = util.train_tune_split(non_tb_dirs, 2)"
```

Comment:
Two quick questions:
1) I don't see a place where you build a training/tuning set for tb_dirs, which is surprising to me. Wouldn't you want both positive and negative examples in your training and tuning data?
2) You split into train and tune with this function, but you also reference test sets in the chunk's comment. Are those split out at a different step or is that comment using a previous terminology?

Reply:
1. Tuberculosis doesn't need a tuning set since the model isn't training on it. The domain adaptation workflow trains a model on sepsis using the tuning set to determine when to stop, then runs the model without further training on tb. Since the function pulling out tb splits at the study level, both tb and sepsis groups will have positive and healthy samples.
2. Good catch! I'm using tb as a test set in the sense that I'm not training the model on it, but the comment is definitely unclear. I Added a commit to fix it.